lldb/test/API/riscv/expressions/TestExpressions.py

```diff
@@ -0,0 +1,96 @@
+"""
+Test RISC-V expressions evaluation.
+"""
```

Comment:
What happens if you run the test-suite on RISC-V? I do think it's good to document what doesn't work, but the successful cases should already be tested elsewhere. So should we just leave the unsupported function calls to test the Arch Pass failures?

Reply:
Yeah I'm not sure if this is known failures or known passes.

Adding skipif riscv or xfail to the known failures seems like the better option. Tip for that, the lldb tests call xfail "expectedfailure".

Reply:
If I'm not mistaken, I've supported all types that frontend can generate (https://github.com/llvm/llvm-project/pull/99336#issuecomment-2340245433), so these tests are no longer relevant and can be removed at all.

However, maybe there are some types I forgot about. Please, let me know if you have any suspicious.

Reply:
Are these tests going to be removed then?

On the grounds that existing tests in the test suite will cover these scenarios.

Reply:
yep, addressed